core/src/main/scala/kafka/admin/ReassignPartitionsCommand.scala

```diff
@@ -138,7 +138,9 @@ object ReassignPartitionsCommand extends Logging {
         // bitwise OR as we don't want to short-circuit
         if (configs.remove(DynamicConfig.Broker.LeaderReplicationThrottledRateProp) != null
           | configs.remove(DynamicConfig.Broker.FollowerReplicationThrottledRateProp) != null
-          | configs.remove(DynamicConfig.Broker.ReplicaAlterLogDirsIoMaxBytesPerSecondProp) != null){
+          | configs.remove(DynamicConfig.Broker.ReplicaAlterLogDirsIoMaxBytesPerSecondProp) != null
+          | configs.remove(DynamicConfig.Broker.LeaderReplicationThrottledProp) != null
```

Comment:
The reassign partitions command doesn't set this config yet so I'm not exactly sure whether we want it to remove it. I think it would be better to have it maintain and rely on the old per-replica configs as that's more specific. I see this new config being set as a more general safety measure that would last throughout reassignments